packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -145,6 +146,14 @@ class MarkdownBuilder implements md.NodeVisitor {
   /// Default tap handler used when [selectable] is set to true
   final VoidCallback? onTapText;
 
+  /// The soft line break pattern is used to identify the spaces at the end of a
+  /// line of text and the leading spaces in the immediately following the line
+  /// of text.
+  ///
+  /// Default these spaces are removed in accordance with the Markdown
+  /// specification on soft line breaks when lines of text are joined.
+  final bool softLineBreakPattern;
```

Comment:
This argument name doesn't make any sense. A `bool` that is the soft line break pattern?

Reply:
What do you think about name change to `softLineBreak`?

Reply:
Sounds good 👍